src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -191,6 +196,53 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
             }
         );
 
+        // update URL parameters according to sort direction UI events
+        // and DO NOT trigger a page refresh
+        this.urlParamsReaction = reaction(
```

Comment:
please merge this one into the other one on line 176

Reply:
The reason for housing the updates in a different reaction is that changes of the sortBy settings do update the URL, but do not trigger the page to reload. If I would house these in the same reaction I would have to do overly complicated evaluation to see which value was updated and trigger a page refresh or url update depending on this.

There was a obvious mistake that I discovered by your comment and that was that the `urlParamsReaction` reaction was overwritten with the reaction for the sortBy params. I have created a new variable ` urlParamsReactionNoRefresh` to remove this collision.